scripts/balancing.lua

```diff
@@ -2017,8 +2017,11 @@ for i,spec in ipairs({
                AddMessage(_("Not enough gold...mine more gold."))
                return false
             end
-            SetUnitVariable(caster, spec.Var, value - 40, "Max")
+            --SetUnitVariable(caster, spec.Var, value - 40, "Max")
             CreateUnit("unit-" .. spec.Unit, ply, {goalX, goalY})
+			if spec.Unit == "brigand" then
```

Comment:
Why are you making two brigands at once?

Reply:
to reduce amount of player attention
if units can be produce every 10 sec, whole game he need to go back to this buildings and keep producing units

if its 2 units every 40 sec, there is more time to focus on something else, plus its more money to accumulate
so this reduce amount of options that burden the player